fairscale/experimental/nn/offload.py

```diff
@@ -386,4 +455,23 @@ def forward(self, *inputs: Any, **_: Any) -> Any:
 
         # We need the second param to be a dummy input to enable the
         # backward pass to be triggered for integer inputs.
-        return ActivationCheckpointing.apply(*inputs, torch.tensor([], requires_grad=True), self)
+        if self._checkpoint_activation:
```

Comment:
oh, I must have reviewed the offending PR and missed that, sorry about that

Reply:
No worries! I realized that the tests weren't really catching this so glad I realized it.